packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/queries.ts

```diff
@@ -712,9 +712,14 @@ export const getStatusWhereClause = (
   return `(${whereClause.join(' OR ')})`;
 };
 
-export const withPagination = (query: string, pagination?: QueryStakePoolsArgs['pagination']) => {
-  if (pagination) return `${query} LIMIT ${pagination.limit} OFFSET ${pagination.startAt} `;
-  return query;
+export const withPagination = (query: string, args: unknown[], pagination?: QueryStakePoolsArgs['pagination']) => {
+  if (pagination) {
+    return {
+      args: [...args, pagination.limit, pagination.startAt],
+      query: `${query} LIMIT $${++args.length} OFFSET $${++args.length}`
```

Comment:
`++Array.length` doesn't sounds as a good idea.
Better to use `Array.length + 1` and `Array.length + 2`

Reply:
0b6d9ee92aaf683c2cfb3d45bd4ff434f354575d